Assets/Scripts/Controllers/Events/TradeController.cs

```diff
@@ -107,9 +107,9 @@ private void TrasfertTradedItems(Trade trade, Vector3 tradingCoordinates)
         {
             if (tradeItem.TradeAmount > 0)
             {
-                Tile tile = WorldController.Instance.World.GetFirstTileWithNoInventoryAround(6, (int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
+                Tile tile = WorldController.Instance.World.GetTileAt((int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
                 Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount, tradeItem.TradeAmount);
-                WorldController.Instance.World.inventoryManager.PlaceInventory(tile, inv);
+                WorldController.Instance.World.inventoryManager.PlaceInventoryAround(tile, inv);
```

Comment:
This officially reduce the search radius from 6 to 3, why?
Taking into account the trade possible quantity and the already lost space because of the landing pad, I disagree, and it it is an issue it shouldn't be part of this PR.